core/src/test/java/org/mskcc/cbio/portal/scripts/TestImportTabDelimData.java

```diff
@@ -83,6 +83,7 @@ public void setUp() throws DaoException {
 		newGeneticProfile.setStableId("study_tcga_pub_test");
 		newGeneticProfile.setProfileName("Barry CNA Results");
 		newGeneticProfile.setDatatype("test");
+		newGeneticProfile.setReferenceGenomeId(1);
```

Comment:
same here

Reply:
reference_id assigned not auto generated. will not change.